File: app/src/main/java/dev/ragnarok/fenrir/link/types/PhotoAlbumLink.java
package dev.ragnarok.fenrir.link.types;

import androidx.annotation.NonNull;

public class PhotoAlbumLink extends AbsLink {

    public final int ownerId;
    public final int albumId;

    public PhotoAlbumLink(int ownerId, int albumId) {
        super(PHOTO_ALBUM);
        this.ownerId = ownerId;
        this.albumId = albumId;
    }

    @NonNull
    @Override
    public String toString() {
        return "PhotoAlbumLink{" +
                "ownerId=" + ownerId +
                ", albumId=" + albumId +
                '}';
    }
}

File: app_fenrir/src/main/kotlin/dev/ragnarok/fenrir/link/types/PhotoAlbumLink.kt
package dev.ragnarok.fenrir.link.types

class PhotoAlbumLink(val ownerId: Long, val albumId: Int) : AbsLink(PHOTO_ALBUM) {
    override fun toString(): String {
        return "PhotoAlbumLink{" +
                "ownerId=" + ownerId +
                ", albumId=" + albumId +
                '}'
    }
}